gammapy/datasets/core.py

```diff
@@ -379,7 +379,7 @@ def copy(self):
         return copy.deepcopy(self)
 
     @classmethod
-    def read(cls, filename, filename_models=None, lazy=True, cache=True):
+    def read(cls, filename, filename_models=None, lazy=True, cache=True, **kwargs):
```

Comment:
I would remove that and ensure that the format is handled correctly by each read/write method.